skillpacks/rating.py

```diff
@@ -0,0 +1,148 @@
+import json
+import time
+from typing import List, Optional, Type
+
+import shortuuid
+from pydantic import BaseModel
+
+from skillpacks.db.conn import WithDB
+from skillpacks.db.models import RatingRecord
+from skillpacks.server.models import ReviewerType, V1Rating
+
+
+class Rating(WithDB):
+    """A numerical rating of an agent action or task"""
+
+    def __init__(
+        self,
+        reviewer: str,
+        rating: int,
```

Comment:
we may just need fields that signify the bounds

Reply:
added!